src/ClientBuilder.php

```diff
@@ -115,7 +116,12 @@
     /**
      * @var string The SDK identifier, to be used in {@see Event} and {@see SentryAuth}
      */
-    private $sdkIdentifier = Client::SDK_IDENTIFIER;
+    private $sdkIdentifier;
+
+    /**
+     * @var string the SDK version of the Client
+     */
+    private $sdkVersion;
```

Comment:
All handling of identifier and version is now moved here, into the `ClientBuilder`